aries-site/src/examples/templates/FilterControls/FiltersContext.js

```diff
@@ -107,6 +117,18 @@ const useFilters = () => {
     return filterResults;
   };
 
+  const getSelected = (array1, array2) => {
```

Comment:
Title this more generically?
```suggestion
  const getIntersection = (array1, array2) => {
```

Reply:
I'm good with that.